README.md

```diff
@@ -80,13 +80,14 @@ To use a Creator DLC the `STEAM_BRANCH` must be set to `creatordlc`
 
 | Name | Flag |
 | ---- | ---- |
-| [CSLA Iron Curtain](https://store.steampowered.com/app/1294440/Arma_3_Creator_DLC_CSLA_Iron_Curtain/) | csla |
-| [Global Mobilization - Cold War Germany](https://store.steampowered.com/app/1042220/Arma_3_Creator_DLC_Global_Mobilization__Cold_War_Germany/) | gm |
+| [CSLA Iron Curtain](https://store.steampowered.com/app/1294440/Arma_3_Creator_DLC_CSLA_Iron_Curtain/) | CSLA |
+| [Global Mobilization - Cold War Germany](https://store.steampowered.com/app/1042220/Arma_3_Creator_DLC_Global_Mobilization__Cold_War_Germany/) | GM |
 | [S.O.G. Prairie Fire](https://store.steampowered.com/app/1227700/Arma_3_Creator_DLC_SOG_Prairie_Fire) | vn |
+| [Western Sahara](https://store.steampowered.com/app/1681170/Arma_3_Creator_DLC_Western_Sahara/) | WS |
 
 ### Example
 
-`-e ARMA_CDLC="csla;gm;vn"`
+`-e ARMA_CDLC="CSLA;GM;vn;WS"`
```

Comment:
These should be lower case

- [CSLA Depot data](https://steamdb.info/depot/233789/)
- [GM Depot data](https://steamdb.info/depot/233787/)
- [VN Depot data](https://steamdb.info/depot/233790/)
- [WS Depot data](https://steamdb.info/depot/233786/)

Reply:
Will update, someone told me they should be uppercase..